mentors/responses.py

```diff
@@ -0,0 +1,20 @@
+get_mentors_200 ={
+  "message": "Mentors Fetched successfully.",
+  "data": [
+    {
+        "id": 0,
+        "name": "string",
+        "email": "string",
+        "contact": "string",
+        "detail": "string",
+        "description": "string",
+        "profile_pic_url": "string",
+        "year": 0,
+        "flag": True

```

Comment:
Provide a real example, not the data types.

Reply:
ok!